src/wp-includes/class-wp-block.php

```diff
@@ -193,6 +193,7 @@ public function __get( $name ) {
 	 */
 	public function render( $options = array() ) {
 		global $post;
+		global $current_parsed_block;
```

Comment:
Minor: This could be one line.

```php
global $post, $current_parsed_block;
```

Reply:
:+1: 

Note that this was part of this fix https://github.com/WordPress/gutenberg/pull/26291 While it works for core, we don't have a way to make it work in the plugin. I'm trying to understand how that can be done but wanted to flag it, in case we need to revisit how block supports work before this patch lands. Related PRs: https://github.com/WordPress/gutenberg/pull/26192 https://github.com/WordPress/gutenberg/pull/26291 https://github.com/WordPress/gutenberg/pull/26282 https://github.com/WordPress/gutenberg/pull/26280

cc @youknowriad @mcsf